src/main/java/net/rubygrapefruit/platform/internal/jni/OsxFileEventFunctions.java

```diff
@@ -60,16 +59,11 @@
      * </ul>
      */
     // TODO How to set kFSEventStreamCreateFlagNoDefer when latency is non-zero?
-    public FileWatcher startWatching(Collection<String> paths, final long latency, final TimeUnit unit, FileWatcherCallback callback) {
-        return createWatcher(paths, callback, new WatcherFactory() {
-            @Override
-            public FileWatcher createWatcher(String[] canonicalPaths, NativeFileWatcherCallback callback) {
-                return startWatching(canonicalPaths, unit.toMillis(latency), callback);
-            }
-        });
+    public FileWatcher startWatcher(long latency, TimeUnit unit, FileWatcherCallback callback) {
```

Comment:
Should this be called `startWatcher`? We start the thread there, though no watching actually starts.

Reply:
That's a good question. It's a bit painful to rename things around here, so I didn't play around with it yet, but I definitely want to clarify the terminology once we have Linux done as well.